config/contract.go

```diff
@@ -45,8 +45,21 @@ log_level: warn
 // DefaultConfigFile is the filename of default config file.
 const DefaultConfigFile = "~/.qingstor/config.yaml"
 
+// EnvConfigPath is config environment variable.
+const EnvConfigPath = "QINGSTOR_CONFIG_PATH"
+
+// EnvAccessKeyID is config envrionment variable.
+const EnvAccessKeyID = "QINGSTOR_ACCESS_KEY_ID"
+
+// EnvSecretAccessKey is config envrionment variable.
+const EnvSecretAccessKey = "QINGSTOR_SECRET_ACCESS_KEY"
+
 // GetUserConfigFilePath returns the user config file path.
 func GetUserConfigFilePath() string {
+	defaultConfigFile := os.Getenv(EnvConfigPath)
+	if defaultConfigFile != "" {
+		return defaultConfigFile
```

Comment:
May be we should convert `~/` to home here?

Reply:
Maybe we should add unit tests here?

Reply:
The environment variable value cannot be ~/


> May be we should convert `~/` to home here?

The environment variable value cannot be ~/

Reply:
> Maybe we should add unit tests here?

OK

Reply:
> 
> > May be we should convert `~/` to home here?
> 
> The environment variable value cannot be ~/
> 

Why？Should we need to return error if we got this input?